devel/libcryptopp/Portfile

```diff
@@ -40,6 +39,12 @@ proc _get_cryptopp_init_priority_flag {compiler} {
     return ""
 }
 
+# https://trac.macports.org/ticket/69597
+if {${configure.cxx_stdlib} eq "libc++"} {
+    patchfiles-append \
+                    patch-libcxx-build.diff
+}
+
 build.type          gnu
 build.target        dynamic static cryptest.exe
 build.env-append    "CXXFLAGS=[get_canonical_archflags cxx] -DNDEBUG [_get_cryptopp_init_priority_flag ${configure.compiler}]"
```

Comment:
Why not just add `${configure.cxxflags}` here?

Reply:
I am not sure what side effects that gonna have.
But yeah, we could do that instead of a patch.

Reply:
@jmroot I have switched to setting cxxflags and dropped the patch.